tests/test_pytorch/test_hpu_graphs.py

```diff
@@ -221,6 +221,7 @@ def test_hpu_graphs(tmpdir, graph_mode, mode):
     train_model(tmpdir, 1, model=model, data_module=data_module, profiler=None, mode=mode)
 
 
+@pytest.mark.skip(reason="TBD: Resolve issues with lightning 2.1")
```

Comment:
```suggestion
@pytest.mark.xfail(reason="TBD: Resolve issues with lightning 2.1")
```
so better know when resolved